packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.11.2+1
 
+* Implement setDescriptionWhileRecording with android camerax (Fixes flutter/flutter#148013).
```

Comment:
This doesn't describe the change to this package. The change for camerax has already landed. This should be something like:

`Adds support to disable persistent recording on Android. See CameraController.startVideoRecording(enablePersistentRecording).`